compiler/circle2circle/src/Circle2Circle.cpp

```diff
@@ -96,6 +96,12 @@ int entry(int argc, char **argv)
       .default_value(false)
       .help("This will convert Custom(Matmul) to Matmul operator");
 
+  arser.add_argument("--fold_dequantize")
```

Comment:
Q) if this his folding, there would be a result of folding. what would the result be?
or is this like remove `dequantize` ?

Reply:
As mentioned in https://github.com/Samsung/ONE/issues/4373#issue-706091190, it is to remove `Dequantize`.

Reply:
If this is remove, could you please use `remove` , `eliminate` or any other words to reflect it's removed?

Reply:
This pass is removing `Dequantize` but main characteristic is folding constant nodes which is used by `Dequantize` by pre-calculating.
If we name it like `RemoveDequantize` or `EliminateDequantize`, it seems that this pass is just removing `Dequantize` without any other actions.
Also, I think that **fold** is reflecting the elimination characteristic too,  just like Constant**Fold** :)

Reply:
OK, thanks for the explanation.